CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this crate are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this crate adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.7.1] - 2024-10-24
```

Comment:
I would probably add these PRs as well:
- https://github.com/paritytech/parity-scale-codec/pull/622
- https://github.com/paritytech/parity-scale-codec/pull/605

Reply:
I was wrong these were already released, missed that 3.7.0 was released recentely